packages/patternfly-4/react-catalog-view-extension/sass/react-catalog-view-extension/_catalog-tile.scss

```diff
@@ -59,26 +59,20 @@
 
 .catalog-tile-pf-description {
   margin-top: 0;
-
-  > span {
-    position: initial;
-  }
 }
 
-.catalog-tile-pf-body {
-  .catalog-tile-pf-description span.truncated {
-    overflow: hidden;
-    position: relative;
+.catalog-tile-pf-description span.truncated {
+  overflow: hidden;
+  position: relative;
 
-    &::after {
-      background: linear-gradient(to right, rgba(255, 255, 255, 0), rgba(255, 255, 255, 1) 75%);
-      bottom: 0;
-      color: transparent;
-      content: ".";
-      position: absolute;
-      right: 0;
-      width: 50%;
-      text-align: right;
-    }
+  & span.truncated::after {
+    background: linear-gradient(to right, rgba(255, 255, 255, 0), rgba(255, 255, 255, 1) 75%);
+    bottom: 0;
+    color: transparent;
+    content: ".";
+    position: absolute;
+    right: 0;
+    width: 50%;
+    text-align: right;
```

Comment:
Better to have:
```
.catalog-tile-pf-description {
 .truncated {
    overflow: hidden;
    position: relative;

    &::after {
      background: linear-gradient(to right, rgba(255, 255, 255, 0), rgba(255, 255, 255, 1) 75%);
      bottom: 0;
      color: transparent;
      content: ".";
      position: absolute;
      right: 0;
      width: 50%;
      text-align: right;
    }
  }
}
```

However, I don't believe we want this only applied when `truncated` is set. The idea is for this to auto truncate when necessary and the caller need not provide truncated text.

Reply:
The class is applied whenever the truncation function is called. The user doesn't have to provide truncated text.

Reply:
Updated the commit with the change.

Reply:
Why the check for `isTruncated` at all?

Reply:
If all the descriptions have these styles, even really short descriptions that don't need truncation get faded out. This is a due to a difference between the old card and the new card. The old card had a fixed height applied based on the amount of space the text took up, so it wasn't an issue. I didn't think it was a great idea to use the fixed height thing with PF4 card styles.

Reply:
OK, this PR is fine by me. The height issue might be a problem if applications try to show tiles with differing header text lengths (causing wrapping). In PF3, that was taken into consideration and the height allowed for the descriptive text auto adjusted for it. Something to watch out for.